pkg/ast/spl/spl.peg

```diff
@@ -576,6 +580,33 @@ Start <- SPACE? initialSearch:(InitialSearchBlock) filterBlocks:(FilterBlock)* q
     return q, nil
 }
 
+IndexAssign <- index:("_index") EQUAL indexName:String {
+    return removeQuotes(indexName), nil
+}
+
+IndexExpression <- first:(IndexAssign) rest:(OR IndexAssign)* {
+    
+    if rest == nil {
+        return first, nil
+    }
+
+    finalIndex := first.(string)
+    for _, v := range rest.([]any) {
+        parts := v.([]any)  // This will be [OR, IndexAssign].
+        index := parts[1].(string)
+        if index == "" {
+            continue
+        }
+        finalIndex = fmt.Sprintf("%s,%s", finalIndex, index)
+    }
+
+    return finalIndex, nil
```

Comment:
I think it will be a little easier to debug/understand if you return a list of strings here, instead of joining the strings with a comma

Reply:
addressed in the latest commit.